flambe/nn/rnn.py

```diff
@@ -83,9 +80,9 @@ def __init__(self,
         self.enforce_sorted = enforce_sorted
         if rnn_type in ['lstm', 'gru']:
             if kwargs:
-                logger.warn(f"The following '{kwargs}' will be ignored " +
-                            "as they are only considered when using 'sru' as " +
-                            "'rnn_type'")
+                warnings.warn(f"The following '{kwargs}' will be ignored " +
```

Comment:
Lets try if this doesn't break our logging mechanism (ie that the warnings appear in our logs)